unstructured/staging/weaviate.py

```diff
@@ -8,7 +8,7 @@ class Properties(TypedDict):
     dataType: List[str]
 
 
-exclude_metadata_keys = ("data_source", "coordinates")
+exclude_metadata_keys = ("data_source", "coordinates", "links", "regex_metadata")
```

Comment:
I wonder if at some point we should switch to a whitelist rather than a blacklist so we don't have to keep changing this.

Reply:
Added #964 to track this one. Was going to just do it real quick but there are enough moving parts to warrant a separate PR.